tldraw/tldraw
# fix: escape eraser tool on escape

Escape the eraser tool on pressing the escape key.

(note that the same bug exists for the laser tool, i'm not sure if the escape key behaviour should be the same for it too)

![licecap-output](https://github.com/tldraw/tldraw/assets/5616556/14bcb1f3-325c-4d28-ad39-a6a7205bd011)

Fixes #1731 

### Change Type


[^1]: publishes a `patch` release, for devDependencies use `internal`
[^2]: will not publish a new version

### Test Plan

1. Select eraser tool
2. Press [escape], you should be back in the select tool
3. Select eraser tool, and start dragging around and press [escape], the behaviour should be same as before


### Release Notes

- escape eraser tool on escape
